db/seeds.rb

```diff
@@ -6,11 +6,11 @@
 end
 
 Dir[
-  File.join(Rails.root, 'db', 'seeders', '**/', '*.rb')
+  File.join(Rails.root, 'db', 'seeders', '*.rb')
```

Comment:
This because you removed the `helpers` subdirectory? So there are no subdirectories that need to be processed?